src/main/java/uk/co/fivium/els/categories/televisions/service/TelevisionsService.java

```diff
@@ -46,6 +46,11 @@ public Document generateHtml(TelevisionsForm form, LegislationCategory legislati
       templatePopulator = new TemplatePopulator(templateParserService.parseTemplate("labels/televisions/televisions-2020.svg"));
     }
 
+    if (form.getPowerSwitch()) {
+      templatePopulator = templatePopulator
```

Comment:
Don't need to reassign the result. The `if` body can just be `templatePopulator.applyCssClassToId("powerSwitch", "hasPowerSwitch");`